core/fmt/fmt.odin

```diff
@@ -44,6 +44,34 @@ Info :: struct {
 // Custom formatter signature. It returns true if the formatting was successful and false when it could not be done
 User_Formatter :: #type proc(fi: ^Info, arg: any, verb: rune) -> bool
 
+// Example User Formatter:
+// SomeType :: struct {
+// 	value: int,
+// }
+// // Custom Formatter for SomeType
+// User_Formatter :: proc(fi: ^fmt.Info, arg: any, verb: rune) -> bool {
+// 	m := cast(^SomeType)arg.data
+// 	switch verb {
+// 	case 'v', 'd':
+// 		fmt.fmt_int(fi, u64(m.value), true, 32, verb)
```

Comment:
Separate but related observation: The `bit_size` argument is a bit confusing. Why is the `bit_size` of a `u64` 32 and not 64?

Reply:
fixed